TheMET/TheMET/ViewControllers/AboutMETViewController.swift

```diff
@@ -29,6 +38,51 @@ class AboutMETViewController: UIViewController {
         )
         aboutButton.tintColor = UIColor(named: "plum")
         self.navigationItem.rightBarButtonItem = aboutButton
+        
+        self.scrollView.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(self.scrollView)
+        NSLayoutConstraint.activate([
+            self.scrollView.leadingAnchor.constraint(equalTo: self.view.leadingAnchor),
+            self.scrollView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor),
+            self.scrollView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor),
+            self.scrollView.topAnchor.constraint(equalTo: self.view.topAnchor)
+        ])
+        let contentGuide = self.scrollView.contentLayoutGuide
+        let frameGuide = self.scrollView.frameLayoutGuide
+        NSLayoutConstraint.activate([
+            contentGuide.leadingAnchor.constraint(equalTo: frameGuide.leadingAnchor),
+            contentGuide.trailingAnchor.constraint(equalTo: frameGuide.trailingAnchor)
+        ])
+        
+        self.emptyView.translatesAutoresizingMaskIntoConstraints = false
+        self.scrollView.addSubview(self.emptyView)
+        NSLayoutConstraint.activate([
+            self.emptyView.heightAnchor.constraint(equalTo: self.view.heightAnchor, multiplier: 0.1),
+            self.emptyView.topAnchor.constraint(equalTo: contentGuide.topAnchor),
+            self.emptyView.leadingAnchor.constraint(equalTo: contentGuide.leadingAnchor),
+            self.emptyView.trailingAnchor.constraint(equalTo: contentGuide.trailingAnchor)
+        ])
+        self.setLocationView(
+            locationView: self.fifthAvenueLocationView, 
+            title: NSLocalizedString("about_met_the_met_fifth_avenue_title_lable", comment: ""),
+            address: NSLocalizedString("about_met_the_met_fifth_avenue_address_lable", comment: ""),
+            topAnchorView: self.emptyView,
+            contentGuide: contentGuide
+        )
+        self.setLocationView(
+            locationView: self.cloistersLocationView,
+            title: NSLocalizedString("about_met_the_met_cloisters_title_lable", comment: ""),
+            address: NSLocalizedString("about_met_the_met_cloisters_address_lable", comment: ""),
+            topAnchorView: self.fifthAvenueLocationView,
+            contentGuide: contentGuide
+        )
+        NSLayoutConstraint.activate([
+            self.cloistersLocationView.bottomAnchor.constraint(equalTo: contentGuide.bottomAnchor)
+        ])
+        let fifthAvenuLocation: CLLocation = CLLocation(latitude: 40.77962342752392, longitude: -73.96326546117187)
```

Comment:
Pls extract this location to some kind of static constants. For example as extension to `CLLocation`